res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1164,6 +1211,11 @@ NumarkMixtrack3.init = function(id,debug) {
     engine.connectControl("[Channel2]", "volume", "NumarkMixtrack3.OnVolumeChange");
     engine.connectControl("[Channel1]", "pfl", "NumarkMixtrack3.OnPFLStatusChange");
     engine.connectControl("[Channel2]", "pfl", "NumarkMixtrack3.OnPFLStatusChange");
+    
+    // The duration control is being used to tell if a track has successfully loaded for smartPFL feature
+    engine.connectControl("[Channel1]", "duration", "NumarkMixtrack3.OnLoadSelectedTrack");
+    engine.connectControl("[Channel2]", "duration", "NumarkMixtrack3.OnLoadSelectedTrack");
```

Comment:
You should use /track_samples/ instead of /duration/ as duration can change when you adjust pitch ubless you want to activate PFL on pitch adjustment.